dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java

```diff
@@ -136,7 +137,8 @@ public class SRTServletResponse implements HttpServletResponse, IResponseOutput,
     private SRTConnectionContext _connContext;
     private IOutputMethodListener outputMethodListener = null;
     private String _contentType;
-
+    private ConcurrentHashMap<Locale,String> localeStrings = new ConcurrentHashMap<Locale, String>();
```

Comment:
This should be static otherwise it will have no value.  Being static it should move to the top of the class by the other static fields.

It should just be new ConcurrentHashMap<>().  No need to provide the types again when using the constructor.

Reply:
Thanks for the review. Made the recommended changes.